incuna-sass/mixins/_em-scale.sass

```diff
@@ -0,0 +1,52 @@
+// Mixin to define a font size scale to allow easier use of ems
+
+// Allows for up to 5 different breakpoints and 6 font sizes
+// Only one breakpoint is required (and two different font sizes)
+
+// The mixin is designed mobile first, so $font-size-1 should be your
+// mobile font size and $breakpoint-1 your smallest breakpoint, then add
+// any remaining required font sizes and breakpoints incrementally
+
+// Example usage:
+
+// .thing
+//     @include em-scale(14px, 'screen and (min-width: 768px)', 16px, 
+//          'screen and (min-width: 1025px)', 20px)
+//     padding: 1em
+
+// the above will create an element with padding that scales to match the
+// font size the font size will change at 2 different breakpoints, first
+// going from 14px to 16px when the screan is 768px wide, and then increasing
+// to 20px at a screen width of 1025px. The padding will scale similarly
+// starting at 14px, then increasing to 16px at the first breakpoint and 20px
+// at the second
+
+// You can also use variables to reference media query variables in your
+// project:
+//     @include em-scale(14px, $px768, 16px, $px1025, 20px)
+// or
+//     @include em-scale(14px, $tablet, 16px, $desktop, 20px)
+
+
+@mixin em-scale($font-size-1, $breakpoint-1, $font-size-2, $breakpoint-2: false, $font-size-3: false, $breakpoint-3: false, $font-size-4: false, $breakpoint-4: false, $font-size-5: false, $breakpoint-5: false, $font-size-6: false)
```

Comment:
I think the numbering is a bit confusing: it seems like the first breakpoint is for the first font-size, but it's actually for the second.

You could rename `$font-size-1` to `$base-font-size` or `$font-size-base`? Then the numbering can be consistent: `$breakpoint-1, $font-size-1, $breakpoint-2, $font-size-2` etc.